### PR TITLE
Fix PEP8 errors

### PR DESCRIPTION
**Pull request by ad-m**
This fixes some of the PEP8 compliance errors.

### CODE REVIEW

**Comment by ad-m** [comment]
In addition, it can cause conflicts, which will result in a new vote.

**Comment by reddraggone9** [comment]
> which will result in a new vote

Nah, that was [fixed](https://github.com/chaosbot/chaos/pull/117).

**Comment by ad-m** [comment]
Oh, you ruined my insidious attack on democracies. 😈

**Comment by ad-m** [comment]
@chaosbot , there is no more conflicts. Stop lying.

**Comment by reddraggone9** [comment]
It caches that for a little while. It takes a couple minutes to update.

File: server/server.py [hunk 1]
```diff
@@ -2,26 +2,34 @@
 import socketserver
 import socket
 
-#set the process name to "chaos_server" so we can easily kill it with "pkill chaos_server"
+# set the process name to "chaos_server" so we can easily kill it with
+# "pkill chaos_server"
```
Comment (reddraggone9) [inline server/server.py:6]: Instead of moving this further away from the function it's documenting, it should probably be made into a docstring.

**Comment by xyproto** [comment]
Fixing Python source files for PEP8 compliance is chasing windmills unless some mechanism is in place for checking and/or fixing source files continously. It's hard just to merge PEP8 related fixes, and PEP8 issues will pop up again and again.

**Comment by reddraggone9** [comment]
#250 will make it easy enough to cause Chaos to stop redeploying if a linter fails if that's the direction we want to go.

**Comment by xyproto** [comment]
:+1: for linting and checking in the modified source files automatically, if there are changes

**Pull request merged**